integration-tests/src/test/java/oracle/kubernetes/operator/ItModelInImageConfigUpdate.java

```diff
@@ -425,6 +425,179 @@ public void testMiiConfigUpdateJdbcImage() {
     LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - " + testMethodName);
   }
 
+  /**
+   * Create a domain without a JDBC DS using model in image and having configmap
+   * in the domain.yaml before deploying the domain. After deploying the domain crd,
+   * re-create the configmap with multiple model files that define a JDBC DataSource.
+   * Each model file overlaps the value of JDBC DS property oracle.net.CONNECT_TIMEOUT
+   * Update the domain crd to change domain restartVersion to reload the model,
+   * generate new config and initiate a rolling restart. The test verifies that loading order
+   * of model files follows WDT rules and the model file loaded last take precedence
+   */
+  @Test
+  public void testMiiMultiModelFilesLoadingOrderName() {
+    Assumptions.assumeTrue(QUICKTEST);
+    String testMethodName = new Object() {
+    }.getClass().getEnclosingMethod().getName();
+    boolean testCompletedSuccessfully = false;
+    final String connTimeout = "5203";
+    Domain domain = null;
+
+    try {
+      logTestBegin(testMethodName);
+      LoggerHelper.getLocal().log(Level.INFO,
+          "Creating Domain & waiting for the script to complete execution");
+
+      // create domain w/o JDBC DS using the image created by MII
+      boolean createDS = false;
+      domain = createDomainUsingMii(createDS);
+      Assertions.assertNotNull(domain, "Failed to create a domain");
+
+      // copy model files that contains JDBC DS to a dir to re-create cm
+      final String destDir = getResultDir() + "/samples/model-in-image-update-lon";
```

Comment:
What does the "-lon" mean?

Reply:
loading order by name :( let me change it to more clearer one

Reply:
fixed